server-adapters/nestjs/src/middleware/json-body.middleware.ts

```
@@ -0,0 +1,45 @@
+
+    this.jsonParser = express.json({
+      limit: maxSize,
+      // Only parse JSON content types
+      type: ['application/json', 'application/*+json'],
+    });
+  }
+
+  use(req: Request, res: Response, next: NextFunction): void {
+    // Skip if body is already parsed (e.g., by another middleware)
+    if (req.body !== undefined && Object.keys(req.body).length > 0) {
+      return next();
```

Biome (2.3.13)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)